### PR TITLE
Update tensorflow version to 1.10.0

### PR DESCRIPTION
**Pull request by kyleabeauchamp**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ requirements:
     - numpy >=1.13.3
     - six >=1.10.0
     - protobuf >=3.4.0
```
Comment (jjhelmus) [inline recipe/meta.yaml]: The protobuf requirement has increased to >=3.6.0

**Comment by kyleabeauchamp** [comment]
Shoot, looks like we need to fix and merge:

https://github.com/conda-forge/libprotobuf-feedstock/pull/10

then 

https://github.com/conda-forge/protobuf-feedstock/pull/52/files

**Comment by kyleabeauchamp** [comment]
It's also unclear to me what the difference is between protobuf and libprotobuf.  They both are installing from the same tarball...

**Comment by jjhelmus** [comment]
I'm working on the update to protobuf.  Should have something available by the end of the day.  

The `libprotobuf` package is the C++ library, `protobuf` is the Python module.  They share the same source code but have different builds.

**Comment by kyleabeauchamp** [comment]
Makes sense.  Thanks!

**Comment by jjhelmus** [comment]
Update to libprotobuf, conda-forge/libprotobuf-feedstock#11

**Comment by jjhelmus** [comment]
protobuf 3.6.0 is now available. I'm closing and restarting this PR to trigger the CIs.

**Comment by dougalsutherland** [comment]
Travis is saying `Failed to load the native TensorFlow runtime.` on `import tensorflow`. Are Google's builds now only supporting newer OS versions than we're using there too?

**Comment by jjhelmus** [comment]
It looks like the Google wheels require Mac OS X 10.12.  From the Travis log:
```
ImportError: dlopen(/Users/travis/miniconda3/conda-bld/tensorflow_1534344511854/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so, 6): Symbol not found: _SecKeyCopyExternalRepresentation
  Referenced from: /Users/travis/miniconda3/conda-bld/tensorflow_1534344511854/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_place/lib/python2.7/site-packages/tensorflow/python/_pywrap_tensorflow_internal.so (which was built for Mac OS X 10.12)
```

**Comment by jjhelmus** [comment]
The official Tensorflow [mac install page](https://www.tensorflow.org/install/install_mac) indicates macOS 10.12.6 (Sierra) or higher is required.

**Comment by dougalsutherland** [comment]
Okay, so we should remove `import tensorflow` from the tests on Mac too, and maybe slightly more strongly consider building it ourselves. :|

**Comment by dougalsutherland** [comment]
@kyleabeauchamp Let's still import on Windows, no?

**Comment by kyleabeauchamp** [comment]
Anyone have any clue which line of the windows test is failing?  The output is not really identifying the failure to me...

**Comment by dougalsutherland** [comment]
Looks like `tensorboard --help` didn't contain the string `USAGE`. Could be for a couple of reasons; maybe try commenting out the `findstr` bit to see the output?

**Comment by kyleabeauchamp** [comment]
Build is green.  Are you ok with just checking that the tensorboard help runs, with no grep?

**Comment by dougalsutherland** [comment]
In older versions, `tensorboard --help` returned 1 (which is why the grep was in there). It looks like that's not the case anymore (thankfully), so this is good.

**Pull request merged**